src/api.rs

```diff
@@ -4,13 +4,19 @@ use log::Event::*;
 use job::{Code, CodeImpl, Job};
 use std::sync::Arc;
 use thread_pool::{self, Registry, WorkerThread};
+use num_cpus;
 
 /// Initializes the Rayon threadpool. You don't normally need to do
 /// this, as it happens automatically, but it is handy for
 /// benchmarking purposes since it avoids initialization overhead in
 /// the actual operations.
-pub fn initialize() {
-    let registry = thread_pool::get_registry();
+/// But you need to call this if you want to specify the maximum number
+/// of threads that Rayon is allowed to use.
+/// If you don't specify it, Rayon uses all the cores avaiable (determined
```

Comment:
My other concern here is that if you call this function *after* a thread-pool has been created (for whatever reason), or if you call it twice, then the value of `num_threads` will be ignored. It's kind of a pain to detect this, though -- so maybe it's enough to just document it.